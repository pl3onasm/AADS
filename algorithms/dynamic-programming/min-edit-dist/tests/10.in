afflictions
failures
5 3 6 8 0 4
