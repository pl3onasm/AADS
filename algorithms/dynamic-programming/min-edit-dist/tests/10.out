Task: source -> target
      'afflictions' -> 'failures'

Costs: copy = 5, insert = 3, delete = 6,
       replace = 8, twiddle = 0, kill = 4

Edit sequence:
  - Source:           'afflictions'
  - Twiddle f and a:  fa-flictions
  - Delete f:         fa-lictions
  - Twiddle i and l:  fail-ctions
  - Insert u:         failu-ctions
  - Insert r:         failur-ctions
  - Insert e:         failure-ctions
  - Insert s:         failures-ctions
  - Kill 06 chars:    failures-
  - Result:           'failures'

Minimum edit distance: 22
